src/abstractdataframe/unique.jl

```diff
@@ -0,0 +1,392 @@
+"""
+    nonunique(df::AbstractDataFrame; keep::Symbol=:first)
+    nonunique(df::AbstractDataFrame, cols; keep::Symbol=:first)
+
+Return a `Vector{Bool}` in which `true` entries indicate duplicate rows.
+
+If `keep=:first` (the default) a row is a duplicate if there exists a prior
+row with all columns containing equal values (according to `isequal`).
+
+If `keep=:last` a row is a duplicate if there exists a subsequent row with all
+columns containing equal values (according to `isequal`).
+
+If `keep=:only` a row is a duplicate if there exists any other row with all
```

Comment:
The term "only" isn't super explicit in this context. I see the name matches the `only` function, but the correspondence isn't perfect, right? I mean, `only` would throw an error if there are duplicates, but here we return `false`.

Maybe a more explicit term would be better? Something like `keep=:noduplicates`? Not sure.

Reply:
I changed it to `:noduplicates`.